src/pycram/helper.py

```diff
@@ -169,3 +169,59 @@ def has(self, index: int) -> bool:
             return True
         except StopIteration:
             return False
+def axis_angle_to_quaternion(axis, angle):
+    """
+    Convert axis-angle to quaternion.
+    axis: (x, y, z) tuple representing rotation axis.
+    angle: rotation angle in degree
+    """
```

Comment:
Please look at the other functions for the doc string. There needs to be a new line between the description and parameter description. Parameter need to be written as :param 'name': ... and return as :return: 
Otherwise the auto generation for the API would not work